sound/soc/codecs/jz4770.c

```diff
@@ -190,12 +190,12 @@ static int jz4770_codec_set_bias_level(struct snd_soc_component *codec,
 
 	switch (level) {
 	case SND_SOC_BIAS_PREPARE:
+		regmap_write(regmap, JZ4770_CODEC_REG_IFR, REG_IFR_ALL_MASK);
 		regmap_clear_bits(regmap, JZ4770_CODEC_REG_CR_VIC,
 				  REG_CR_VIC_SB);
 		msleep(250);
 		regmap_clear_bits(regmap, JZ4770_CODEC_REG_CR_VIC,
 				  REG_CR_VIC_SB_SLEEP);
-		msleep(400);
```

Comment:
Why is that one removed?

Reply:
To be in line with the documentation recommendation : 
When SB_SLEEP falls, CODEC leaves the corresponding STANDBY mode; it is necessary to wait
some time before the CODEC reaches the normal mode. Depending on the selected mode, this time is
either called Tshd_adc (SB_ADC=0) for the ADC path or Tshd_dac (SB_DAC=0) for the DAC path.

Tshd_dac Typ 400 Max 900 is thus supposed to happen after SB_DAC=0 and isn't needed after SB_SLEEP=0 imho, this is simply subject to review I have no hard opinion.